src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -39,7 +39,9 @@ can also be internationalized using the same classes and logic.
   * [Internationalization based on the `intl` package][]
 {{site.alert.end}}
 
-## Setting up an internation&shy;alized app: the flutter<wbr>_localizations package {#setting-up}
+## Introduction to Localizations in Flutter
+
```

Comment:
The intro text to this page says that this is a "tutorial" on how to localize your app, but it doesn't feel like a tutorial. It feels like more of a deep dive into how localization works, it's more of a conceptual doc.

I'm wondering if you can add some text up front. Sort of a summary on how to localize. Can it be boiled down into a few steps? Like:

You can localize your app in a number of ways, but here is a quick-start guide:

1. Add the localizations package to your pubspec.yaml file.
2. Choose the languages you want to support. For languages that include variants, specify the variants you want to support, such as French _and_ Canadian French (fr_FR, fr_CA).
3. For iOS, ...
4. For Android,...
5. For web...

Also, are you planning to add the new doc to this same page? Or will it be a secondary page?
6. For desktop...

The way the doc is now, there is a LOT of text and the steps feel fairly buried. I just want to know what steps to follow. I can later dive in deeper, if I want.

Reply:
Right -- I'll rewrite the intro text to the page, since it isn't a tutorial. From my understanding, there are two ideas we'll want our readers to immediately take away:

1. How to localize the Cupertino and Material libraries. This basically covers the widgets we've already localized and offer out of the box for Flutter.
2. How to create localizations for your own custom widgets, which is for everything else that they need to localize. This is what the new l10n tool that Hans and I have been working on accomplishes.

What I'm thinking is I could write a "quick-start" tutorial that does both, since the steps are fairly intertwined.

Reply:
A "quick-start" guide would be fabulous!!!